doc/whats_new.rst

```diff
@@ -29,6 +29,8 @@ Changelog
 
 - Update plot_hist_input() to plot_spikes_hist() which can plot histogram of spikes for any cell type, by `Nick Tolley`_ in `#157 <https://github.com/jonescompneurolab/hnn-core/pull/157>`_
 
+- Add function to compute mean spike rates with user specified calculation type, by `Nick Tolley` and `Mainak Jas`_ in `#155 <https://github.com/jonescompneurolab/hnn-core/pull/155>`_
```

Comment:
your name won't have the hyperlink. Do you know how to build the documentation locally? You need to run:

$ make no-html

from inside the doc/ folder